README.md

```diff
@@ -69,6 +69,14 @@ Prebuilt EOSIO software packages are available for the operating systems below.
 brew tap eosio/eosio
 brew install eosio
 ```
+Note: On MacOS 10.15 (Catalina), there is a chance to face the error below which prevents successful installation of Eosio:
```

Comment:
"Eosio" is stylized in all-uppercase, "EOSIO". We can also be more specific, that this is a linking error. What about this?
```md
Note: On MacOS 10.15 (Catalina), there is a chance to face the linking error below which prevents successful installation of EOSIO:
```

Reply:
Done.